templates/pcab/page.tpl.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+    <head>
+        <link href="css\pcab.css" type="text/css" rel="stylesheet">
+        <meta charset="utf-8"/>
+        <title><%= title %></title>
+    </head>
+    <body>
+        <%= header.html %>
+        <main>
+            <section class="intro">
+                    <h1><%= title %></h1>
+                        <%= imageHtml %>
```

Comment:
The indentation looks off for the `imageHtml` row. This is probably wrong in `default/page.tpl.html` as well